runtime/moonbase/tests/integration_test.rs

```diff
@@ -16,15 +16,12 @@
 
 //! Moonbase Runtime Integration Tests
 
-#![cfg(test)]
```

Comment:
I still think this should be here and it should be added to the top of `common.rs` as well

Reply:
@tgmichel was this resolved because you will add it or because it is already covered somewhere else. All the code in `tests/` should only be compiled on test.

Reply:
Sorry! I was convinced I wrote the response. As described in the book:

> We don’t need to annotate any code in tests/integration_test.rs with #[cfg(test)]

Reply:
Cool, cargo knows that any files in the `tests` folder will only be `#[cfg(test)]`